index.html

```diff
@@ -79,34 +91,43 @@ <h2 class="Tonic">Tonic</h2>
                 <button class="button">See Project</button>
             </div>
         </div>
-        <div class="card-work-2">
+        <div class="card-work-2 work2">
             <div class="work-image-3"></div>
             <div class="left-block">
-                <p class="Tonic">Tonic</p>
+                <p class="Tonic one">Tonic</p>
+                <p class="Tonic two">Facebook 360</p>
                 <div class="frame1">
                     <p class="canopy">CANOPY</p>
                     <p class="back-end-dev">Back End Dev</p>
                     <p class="year">2015</p>
                 </div>
-                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="desc one">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <p class="desc two">Exploring the future of media in Facebook's first Virtual Reality app; a place to
+                    discover and enjoy 360 photos and videos on Gear VR.</p>
                 <div class="tags">
                     <a class="html">html</a>
                     <a class="css">css</a>
                     <a class="javascript">javascript</a>
                 </div>
                 <button class="button">See Project</button>
             </div>
+            <div class="hand-cursor"></div>
         </div>
         <div class="card-work-2">
             <div class="work-image-4"></div>
             <div class="left-block">
-                <p class="Tonic">Multi-Post <br> Stories</p>
+                <p class="Tonic one">Multi-Post <br> Stories</p>
+                <p class="Tonic two">Uber Navigation</p>
                 <div class="frame1">
                     <p class="canopy">CANOPY</p>
                     <p class="back-end-dev">Back End Dev</p>
                     <p class="year">2015</p>
                 </div>
-                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="desc one">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <p class="desc two">A smart assistant to make driving more safe, efficient, and fun by unlocking your
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you sue the correct padding and margin for spacing your elements on the page. 
Your design :point_down: 
![Screenshot from 2022-09-30 15-58-48](https://user-images.githubusercontent.com/97100091/193287209-5bf551f2-8c37-4319-b0d0-b410617142f9.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193287526-e730266b-b2e2-4983-9925-77fc547500b3.png)

Reply:
[OPTIONAL]
- [ ] Kindly make sure also you implement the button interaction `hover`, `active`, and `disable` as shown in Figma